### ISSUE TITLE
QRCodeDetector::detectAndDecode doesn't decode

### ISSUE THREAD

**Issue by Yyote** [opencv/opencv#24346]
### System Information

OpenCV version: 4.8.0
Operating System / Platform: Ubuntu 20.04
Compiler & compiler version: GCC 9.4.0

### Detailed description

I create a `QRCodeDetector` and try to detect and decode the QR code using `detectAndDecode()` method:

```
QRCodeDetector qr_detect;
std::string qr_string = qr_detect.detectAndDecode(frame, points, frame2);
```

It detects the QR but it doesn't decode it and prints this in terminal:
```
Library QUIRC is not linked. No decoding is performed. Take it to the OpenCV repository.
```

### Steps to reproduce

```cpp
#include <opencv2/objdetect.hpp>
#include <opencv2/imgcodecs.hpp>
#include <opencv2/highgui/highgui.hpp>
#include <opencv2/imgproc/imgproc.hpp>
#include <iostream>
#include "opencv2/opencv.hpp"
#include "opencv2/aruco.hpp"


int main()
{
    cv::Mat frame;
    frame = cv::imread("/home/leev/drone_ws/src/packageInDev/opencv_projects/aruco_detection/imgs/QR.png");

    // print_flag(1);
    CV_Assert(frame.depth() == CV_8U);
    CV_Assert(frame.channels() == 3);

    // print_flag(1);
    int nRows = frame.rows;
    int nCols = frame.cols;

    // print_flag(1);
    int center_rows = int(nRows / 2);
    int center_cols = int(nCols / 2);

    while(true)
    {
        cv::Mat frame2 = frame.clone();
        std::vector<std::vector<cv::Point2f> > corners;
        std::vector<int> ids;
        std::vector<cv::Point> points;

        cv::QRCodeDetector qr_detect;
        std::string qr_string = qr_detect.detectAndDecode(frame, points, frame2);
        
        cv::imshow("objects", frame);

        cv::line(frame2, points.at(0), points.at(1), cv::Scalar(255, 0, 0), 5);
        cv::line(frame2, points.at(1), points.at(2), cv::Scalar(255, 0, 0), 5);
        cv::line(frame2, points.at(2), points.at(3), cv::Scalar(255, 0, 0), 5);
        cv::line(frame2, points.at(3), points.at(0), cv::Scalar(255, 0, 0), 5);

        if (!frame2.empty())
        {
            // cvtColor(frame2, frame2, COLOR_GRAY2BGR);
            cv::imshow("objects2", frame2);
        }

        std::cout << qr_string;
        std::cout << points.size() << std::endl;

        if( cv::waitKey(5) >= 0 ) 
        {
            break; // stop capturing by pressing ESC 
        }
    }
    std::cout << "nRows: " << nRows << "\nnCols: " << nCols;
    return 0;
}
```

### Issue submission checklist

- [X] I report the issue, it's not a question
- [X] I checked the problem with documentation, FAQ, open issues, forum.opencv.org, Stack Overflow, etc and have not found any solution
- [X] I updated to the latest OpenCV version and the issue is still there
- [X] There is reproducer code and related data files (videos, images, onnx, etc)

**Comment by Kumataro** [comment]
Hello, I expected that your opencv library has no `quirk` library because you set `-DWITH_QUIRC=OFF` at executing cmake to create Makefile.

Please could you share your `cmake` command options and output log to compile opencv library?

And please could you check `3rdparty dependencies:` line ?
If it has no `quirc`, it means that `Library QUIRC is not linked`.

```bash
$ cmake ..


-- General configuration for OpenCV 4.8.0 =====================================
--   Version control:               4.x.0-xxx-xxxxxxxxxx
--
--   Platform:


--   C/C++:
--     Built as dynamic libs?:      NO
--     C++ standard:                11


--     Extra dependencies:          ade dl m pthread rt
--     3rdparty dependencies:       ittnotify libprotobuf zlib libjpeg-turbo libwebp libpng libtiff libjasper quirc
```

**Comment by dkurt** [comment]
Decoding without Quirc dependency is also in progress: https://github.com/opencv/opencv/pull/24299

**Comment by Yyote** [comment]
> Hello, I expected that your opencv library has no `quirk` library because you set `-DWITH_QUIRC=OFF` at executing cmake to create Makefile.
> 
> Please could you share your `cmake` command options and output log to compile opencv library?
> 
> And please could you check `3rdparty dependencies:` line ? If it has no `quirc`, it means that `Library QUIRC is not linked`.

Here are the `cmake` command options:
```
sudo cmake -DOPENCV_EXTRA_MODULES_PATH=../opencv_contrib-4.x/modules ../opencv-4.x
```

`3rdparty dependencies:` line is empty.

Output log: 
```
-- General configuration for OpenCV 4.8.0-dev =====================================
--   Version control:               unknown
-- 
--   Extra modules:
--     Location (extra):            /home/leev/opencv_contrib-4.x/modules
--     Version control (extra):     unknown
-- 
--   Platform:
--     Timestamp:                   2023-09-30T12:35:13Z
--     Host:                        Linux 5.15.0-84-generic x86_64
--     CMake:                       3.16.3
--     CMake generator:             Unix Makefiles
--     CMake build tool:            /usr/bin/make
--     Configuration:               Release
-- 
--   CPU/HW features:
--     Baseline:                    SSE SSE2 SSE3
--       requested:                 SSE3
--     Dispatched code generation:  SSE4_1 SSE4_2 FP16 AVX AVX2 AVX512_SKX
--       requested:                 SSE4_1 SSE4_2 AVX FP16 AVX2 AVX512_SKX
--       SSE4_1 (18 files):         + SSSE3 SSE4_1
--       SSE4_2 (2 files):          + SSSE3 SSE4_1 POPCNT SSE4_2
--       FP16 (1 files):            + SSSE3 SSE4_1 POPCNT SSE4_2 FP16 AVX
--       AVX (9 files):             + SSSE3 SSE4_1 POPCNT SSE4_2 AVX
--       AVX2 (38 files):           + SSSE3 SSE4_1 POPCNT SSE4_2 FP16 FMA3 AVX AVX2
--       AVX512_SKX (8 files):      + SSSE3 SSE4_1 POPCNT SSE4_2 FP16 FMA3 AVX AVX2 AVX_512F AVX512_COMMON AVX512_SKX
-- 
--   C/C++:
--     Built as dynamic libs?:      YES
--     C++ standard:                11
--     C++ Compiler:                /usr/bin/c++  (ver 9.4.0)
--     C++ flags (Release):         -fsigned-char -W -Wall -Wreturn-type -Wnon-virtual-dtor -Waddress -Wsequence-point -Wformat -Wformat-security -Wmissing-declarations -Wundef -Winit-self -Wpointer-arith -Wshadow -Wsign-promo -Wuninitialized -Wsuggest-override -Wno-delete-non-virtual-dtor -Wno-comment -Wimplicit-fallthrough=3 -Wno-strict-overflow -fdiagnostics-show-option -Wno-long-long -pthread -fomit-frame-pointer -ffunction-sections -fdata-sections  -msse -msse2 -msse3 -fvisibility=hidden -fvisibility-inlines-hidden -O3 -DNDEBUG  -DNDEBUG
--     C++ flags (Debug):           -fsigned-char -W -Wall -Wreturn-type -Wnon-virtual-dtor -Waddress -Wsequence-point -Wformat -Wformat-security -Wmissing-declarations -Wundef -Winit-self -Wpointer-arith -Wshadow -Wsign-promo -Wuninitialized -Wsuggest-override -Wno-delete-non-virtual-dtor -Wno-comment -Wimplicit-fallthrough=3 -Wno-strict-overflow -fdiagnostics-show-option -Wno-long-long -pthread -fomit-frame-pointer -ffunction-sections -fdata-sections  -msse -msse2 -msse3 -fvisibility=hidden -fvisibility-inlines-hidden -g  -O0 -DDEBUG -D_DEBUG
--     C Compiler:                  /usr/bin/cc
--     C flags (Release):           -fsigned-char -W -Wall -Wreturn-type -Waddress -Wsequence-point -Wformat -Wformat-security -Wmissing-declarations -Wmissing-prototypes -Wstrict-prototypes -Wundef -Winit-self -Wpointer-arith -Wshadow -Wuninitialized -Wno-comment -Wimplicit-fallthrough=3 -Wno-strict-overflow -fdiagnostics-show-option -Wno-long-long -pthread -fomit-frame-pointer -ffunction-sections -fdata-sections  -msse -msse2 -msse3 -fvisibility=hidden -O3 -DNDEBUG  -DNDEBUG
--     C flags (Debug):             -fsigned-char -W -Wall -Wreturn-type -Waddress -Wsequence-point -Wformat -Wformat-security -Wmissing-declarations -Wmissing-prototypes -Wstrict-prototypes -Wundef -Winit-self -Wpointer-arith -Wshadow -Wuninitialized -Wno-comment -Wimplicit-fallthrough=3 -Wno-strict-overflow -fdiagnostics-show-option -Wno-long-long -pthread -fomit-frame-pointer -ffunction-sections -fdata-sections  -msse -msse2 -msse3 -fvisibility=hidden -g  -O0 -DDEBUG -D_DEBUG
--     Linker flags (Release):      -Wl,--exclude-libs,libippicv.a -Wl,--exclude-libs,libippiw.a   -Wl,--gc-sections -Wl,--as-needed -Wl,--no-undefined  
--     Linker flags (Debug):        -Wl,--exclude-libs,libippicv.a -Wl,--exclude-libs,libippiw.a   -Wl,--gc-sections -Wl,--as-needed -Wl,--no-undefined  
--     ccache:                      YES
--     Precompiled headers:         NO
--     Extra dependencies:          dl m pthread rt
--     3rdparty dependencies:
-- 
--   OpenCV modules:
--     To be built:                 alphamat aruco bgsegm bioinspired calib3d ccalib core datasets dnn dnn_objdetect dnn_superres dpm face features2d flann freetype fuzzy gapi hdf hfs highgui img_hash imgcodecs imgproc intensity_transform java line_descriptor mcc ml objdetect optflow phase_unwrapping photo plot python3 quality rapid reg rgbd saliency shape stereo stitching structured_light superres surface_matching text tracking ts video videoio videostab viz wechat_qrcode xfeatures2d ximgproc xobjdetect xphoto
--     Disabled:                    world
--     Disabled by dependency:      -
--     Unavailable:                 cudaarithm cudabgsegm cudacodec cudafeatures2d cudafilters cudaimgproc cudalegacy cudaobjdetect cudaoptflow cudastereo cudawarping cudev cvv julia matlab ovis python2 sfm
--     Applications:                tests perf_tests apps
--     Documentation:               NO
--     Non-free algorithms:         NO
-- 
--   GUI:                           GTK2
--     GTK+:                        YES (ver 2.24.32)
--       GThread :                  YES (ver 2.64.6)
--       GtkGlExt:                  NO
--     VTK support:                 YES (ver 7.1.1)
-- 
--   Media I/O: 
--     ZLib:                        /usr/lib/x86_64-linux-gnu/libz.so (ver 1.2.11)
--     JPEG:                        /usr/lib/x86_64-linux-gnu/libjpeg.so (ver 80)
--     WEBP:                        /usr/lib/x86_64-linux-gnu/libwebp.so (ver encoder: 0x020e)
--     PNG:                         /usr/lib/x86_64-linux-gnu/libpng.so (ver 1.6.37)
--     TIFF:                        /usr/lib/x86_64-linux-gnu/libtiff.so (ver 42 / 4.1.0)
--     JPEG 2000:                   OpenJPEG (ver 2.3.1)
--     OpenEXR:                     /usr/lib/x86_64-linux-gnu/libImath.so /usr/lib/x86_64-linux-gnu/libIlmImf.so /usr/lib/x86_64-linux-gnu/libIex.so /usr/lib/x86_64-linux-gnu/libHalf.so /usr/lib/x86_64-linux-gnu/libIlmThread.so (ver 2_3)
--     HDR:                         YES
--     SUNRASTER:                   YES
--     PXM:                         YES
--     PFM:                         YES
-- 
--   Video I/O:
--     DC1394:                      YES (2.2.5)
--     FFMPEG:                      YES
--       avcodec:                   YES (58.54.100)
--       avformat:                  YES (58.29.100)
--       avutil:                    YES (56.31.100)
--       swscale:                   YES (5.5.100)
--       avresample:                YES (4.0.0)
--     GStreamer:                   YES (1.16.3)
--     v4l/v4l2:                    YES (linux/videodev2.h)
-- 
--   Parallel framework:            pthreads
-- 
--   Trace:                         YES (with Intel ITT)
-- 
--   Other third-party libraries:
--     Intel IPP:                   2021.8 [2021.8.0]
--            at:                   /home/leev/build/3rdparty/ippicv/ippicv_lnx/icv
--     Intel IPP IW:                sources (2021.8.0)
--               at:                /home/leev/build/3rdparty/ippicv/ippicv_lnx/iw
--     VA:                          NO
--     Lapack:                      NO
--     Eigen:                       YES (ver 3.3.7)
--     Custom HAL:                  NO
--     Protobuf:                    build (3.19.1)
--     Flatbuffers:                 builtin/3rdparty (23.5.9)
-- 
--   OpenCL:                        YES (no extra features)
--     Include path:                /home/leev/opencv-4.x/3rdparty/include/opencl/1.2
--     Link libraries:              Dynamic load
-- 
--   Python 3:
--     Interpreter:                 /usr/bin/python3 (ver 3.8.10)
--     Libraries:                   /usr/lib/x86_64-linux-gnu/libpython3.8.so (ver 3.8.10)
--     numpy:                       /home/leev/.local/lib/python3.8/site-packages/numpy/core/include (ver 1.24.4)
--     install path:                lib/python3.8/site-packages/cv2/python-3.8
-- 
--   Python (for build):            /usr/bin/python2.7
-- 
--   Java:                          
--     ant:                         NO
--     Java:                        YES (ver 13.0.7)
--     JNI:                         /usr/lib/jvm/java-13-openjdk-amd64/include /usr/lib/jvm/java-13-openjdk-amd64/include/linux /usr/lib/jvm/java-13-openjdk-amd64/include
--     Java wrappers:               YES (JAVA)
--     Java tests:                  NO
-- 
--   Install to:                    /usr/local
-- -----------------------------------------------------------------
```

**Comment by Kumataro** [comment]
Thank you for your reply! 
My comment is for 4.8.0 release code.
However you seems to try with 4.x branch code.
So there are small differences.

( I'm sorry that it is repeat about https://github.com/opencv/opencv/issues/24346#issuecomment-1742116669  ).
In 4.x branch, in-house QR decoder is fixing now at https://github.com/opencv/opencv/pull/24299

Please could you wait to finish this fix or use 4.8.1 release ?

**Comment by Kumataro** [comment]
Hello, I continued to investigate this problem more.

I tried a very simple test, and it works well with current 4.x branch( https://github.com/opencv/opencv/commit/63819c1e1f7b8880ff55a1773d4219f431ee3cf7).
Maybe trying the latest code will solve your problem.


1. Could you re-try with update recent code from `git clone` ?
It seems that you try with zipped source code, however it is hard to detect what version did you use. I believe that using git is helpful for us. 
Because git commit hashcode is left on cmake log.

> git clone https://github.com/opencv/opencv.git
> git -C opencv checkout 4.x

2. If your working folder is not empty at executing cmake, please make new empty working folder.  CMake left some settings at CMakeCache.txt/CMakeVars.txt and reuse it.

for example, following code is creating worling folder automatically.

````bash
cmake -S opencv -B build4-main 
````

-----------------------------------------------------------------

Following information is successed with my environment.

```bash
commit 63819c1e1f7b8880ff55a1773d4219f431ee3cf7 (HEAD -> 4.x, origin/HEAD, origin/4.x)
Merge: f3c724d449 2af5815d47
Author: Alexander Smorkalov <2536374+asmorkalov@users.noreply.github.com>
Date:   Mon Oct 2 09:17:11 2023 +0300

    Merge pull request #24342 from asmorkalov:as/java_test_status

    Fail Java test suite, execution, if one of test failed.
```

```
kmtr@ubuntu:~/work/build4-main$ ./bin/opencv_version -v

General configuration for OpenCV 4.8.0-dev =====================================
  Version control:               4.8.0-278-g63819c1e1f

  Platform:
    Timestamp:                   2023-10-02T11:47:46Z
    Host:                        Linux 5.15.0-76-generic x86_64
    CMake:                       3.16.3
    CMake generator:             Ninja
    CMake build tool:            /usr/bin/ninja
    Configuration:               Release

  CPU/HW features:
    Baseline:                    SSE SSE2 SSE3
      requested:                 SSE3
    Dispatched code generation:  SSE4_1 SSE4_2 FP16 AVX AVX2 AVX512_SKX
      requested:                 SSE4_1 SSE4_2 AVX FP16 AVX2 AVX512_SKX
      SSE4_1 (14 files):         + SSSE3 SSE4_1
      SSE4_2 (1 files):          + SSSE3 SSE4_1 POPCNT SSE4_2
      FP16 (0 files):            + SSSE3 SSE4_1 POPCNT SSE4_2 FP16 AVX
      AVX (3 files):             + SSSE3 SSE4_1 POPCNT SSE4_2 AVX
      AVX2 (28 files):           + SSSE3 SSE4_1 POPCNT SSE4_2 FP16 FMA3 AVX AVX2
      AVX512_SKX (3 files):      + SSSE3 SSE4_1 POPCNT SSE4_2 FP16 FMA3 AVX AVX2 AVX_512F AVX512_COMMON AVX512_SKX

  C/C++:
    Built as dynamic libs?:      YES
    C++ standard:                11
    C++ Compiler:                /usr/bin/c++  (ver 9.4.0)
    C++ flags (Release):         -fsigned-char -W -Wall -Wreturn-type -Wnon-virtual-dtor -Waddress -Wsequence-point -Wformat -Wformat-security -Wmissing-declarations -Wundef -Winit-self -Wpointer-arith -Wshadow -Wsign-promo -Wuninitialized -Wsuggest-override -Wno-delete-non-virtual-dtor -Wno-comment -Wimplicit-fallthrough=3 -Wno-strict-overflow -fdiagnostics-show-option -Wno-long-long -pthread -fomit-frame-pointer -ffunction-sections -fdata-sections  -msse -msse2 -msse3 -fvisibility=hidden -fvisibility-inlines-hidden -O3 -DNDEBUG  -DNDEBUG
    C++ flags (Debug):           -fsigned-char -W -Wall -Wreturn-type -Wnon-virtual-dtor -Waddress -Wsequence-point -Wformat -Wformat-security -Wmissing-declarations -Wundef -Winit-self -Wpointer-arith -Wshadow -Wsign-promo -Wuninitialized -Wsuggest-override -Wno-delete-non-virtual-dtor -Wno-comment -Wimplicit-fallthrough=3 -Wno-strict-overflow -fdiagnostics-show-option -Wno-long-long -pthread -fomit-frame-pointer -ffunction-sections -fdata-sections  -msse -msse2 -msse3 -fvisibility=hidden -fvisibility-inlines-hidden -g  -O0 -DDEBUG -D_DEBUG
    C Compiler:                  /usr/bin/cc
    C flags (Release):           -fsigned-char -W -Wall -Wreturn-type -Waddress -Wsequence-point -Wformat -Wformat-security -Wmissing-declarations -Wmissing-prototypes -Wstrict-prototypes -Wundef -Winit-self -Wpointer-arith -Wshadow -Wuninitialized -Wno-comment -Wimplicit-fallthrough=3 -Wno-strict-overflow -fdiagnostics-show-option -Wno-long-long -pthread -fomit-frame-pointer -ffunction-sections -fdata-sections  -msse -msse2 -msse3 -fvisibility=hidden -O3 -DNDEBUG  -DNDEBUG
    C flags (Debug):             -fsigned-char -W -Wall -Wreturn-type -Waddress -Wsequence-point -Wformat -Wformat-security -Wmissing-declarations -Wmissing-prototypes -Wstrict-prototypes -Wundef -Winit-self -Wpointer-arith -Wshadow -Wuninitialized -Wno-comment -Wimplicit-fallthrough=3 -Wno-strict-overflow -fdiagnostics-show-option -Wno-long-long -pthread -fomit-frame-pointer -ffunction-sections -fdata-sections  -msse -msse2 -msse3 -fvisibility=hidden -g  -O0 -DDEBUG -D_DEBUG
    Linker flags (Release):      -Wl,--exclude-libs,libippicv.a -Wl,--exclude-libs,libippiw.a   -Wl,--gc-sections -Wl,--as-needed -Wl,--no-undefined
    Linker flags (Debug):        -Wl,--exclude-libs,libippicv.a -Wl,--exclude-libs,libippiw.a   -Wl,--gc-sections -Wl,--as-needed -Wl,--no-undefined
    ccache:                      YES
    Precompiled headers:         NO
    Extra dependencies:          dl m pthread rt
    3rdparty dependencies:

  OpenCV modules:
    To be built:                 calib3d core features2d flann imgcodecs imgproc objdetect
    Disabled:                    dnn gapi highgui java_bindings_generator js_bindings_generator ml objc_bindings_generator photo python_bindings_generator python_tests stitching ts video videoio world
    Disabled by dependency:      -
    Unavailable:                 java python2 python3
    Applications:                apps
    Documentation:               NO
    Non-free algorithms:         NO

  GUI:
    GTK+:                        YES (ver 3.24.20)
      GThread :                  YES (ver 2.64.6)
      GtkGlExt:                  NO
    VTK support:                 NO

  Media I/O:
    ZLib:                        /usr/lib/x86_64-linux-gnu/libz.so (ver 1.2.11)
    JPEG:                        libjpeg-turbo (ver 2.1.3-62)
    WEBP:                        build (ver encoder: 0x020f)
    PNG:                         /usr/lib/x86_64-linux-gnu/libpng.so (ver 1.6.37)
    TIFF:                        build (ver 42 - 4.2.0)
    JPEG 2000:                   build (ver 2.5.0)
    OpenEXR:                     build (ver 2.3.0)
    HDR:                         YES
    SUNRASTER:                   YES
    PXM:                         YES
    PFM:                         YES

  Video I/O:
    DC1394:                      NO
    FFMPEG:                      NO
      avcodec:                   NO
      avformat:                  NO
      avutil:                    NO
      swscale:                   NO
      avresample:                NO
    GStreamer:                   NO
    v4l/v4l2:                    YES (linux/videodev2.h)

  Parallel framework:            pthreads

  Trace:                         YES (with Intel ITT)

  Other third-party libraries:
    Intel IPP:                   2021.8 [2021.8.0]
           at:                   /home/kmtr/work/build4-main/3rdparty/ippicv/ippicv_lnx/icv
    Intel IPP IW:                sources (2021.8.0)
              at:                /home/kmtr/work/build4-main/3rdparty/ippicv/ippicv_lnx/iw
    VA:                          NO
    Lapack:                      NO
    Eigen:                       NO
    Custom HAL:                  NO
    Protobuf:                    build (3.19.1)
    Flatbuffers:                 builtin/3rdparty (23.5.9)

  OpenCL:                        YES (no extra features)
    Include path:                /home/kmtr/work/opencv/3rdparty/include/opencl/1.2
    Link libraries:              Dynamic load

  Python (for build):            /usr/bin/python3

  Java:
    ant:                         NO
    Java:                        NO
    JNI:                         NO
    Java wrappers:               NO
    Java tests:                  NO

  Install to:                    /usr/local
-----------------------------------------------------------------
```

libopencv_objdetect has some `quirc_*` functions.

```bash
kmtr@ubuntu:~/work/build4-main$ nm lib/libopencv_objdetect.so  | c++filt  | grep quirc
00000000000bf7f0 t quirc_decode
00000000000c2670 t quirc_flip
00000000000d9920 r quirc_version_db
```


test data is from https://github.com/opencv/opencv_extra/blob/4.x/testdata/cv/qrcode/link_github_ocv.jpg

```cpp
#include <opencv2/core.hpp>
#include <opencv2/imgcodecs.hpp>
#include <opencv2/objdetect.hpp>
#include <iostream>
#include <string>

int main()
{
    cv::Mat frame = cv::imread("link_github_ocv.jpg", cv::IMREAD_COLOR);

    int nRows = frame.rows;
    int nCols = frame.cols;

    cv::Mat frame2 = frame.clone();
    std::vector<std::vector<cv::Point2f> > corners;
    std::vector<int> ids;
    std::vector<cv::Point> points;

    cv::QRCodeDetector qr_detect;
    std::string qr_string = qr_detect.detectAndDecode(frame, points, frame2);

    std::cout << "string = " << qr_string << std::endl;
    std::cout << "points = " << points << std::endl;

    return 0;
}
```

```Makefile
a.out : main.cpp
        g++ main.cpp -o a.out \
                -I/usr/local/include/opencv4 \
                -lopencv_core \
                -lopencv_imgcodecs \
                -lopencv_objdetect
```

```bash
$ ./a.out
string = https://github.com/opencv/opencv
points = [484, 97;
 378, 251;
 189, 138;
 311, 8]
```
